components/tilePlayer.tsx

```diff
@@ -1,81 +1,101 @@
-import React, { useRef, useState } from 'react';
-import ReactPlayer from 'react-player';
-import { AudioPlayer, ImageContainer, Controls, SliderContainer, StyledSlider } from './tilePlayer.style';
+import React, { useRef, useState, useEffect } from 'react'
+import ReactPlayer from 'react-player'
+import {
+  AudioPlayer,
+  ImageContainer,
+  Controls,
+  SliderContainer,
+  StyledSlider,
+} from './tilePlayer.style'
 
 interface TilePlayerProps {
-  src: string;
-  imageSrc: string;
+  src: string
+  imageSrc: string
+  incrementActiveSounds: (boolean) => void
```

Comment:
> 14 |   incrementActiveSounds: (boolean) => void

TS complains that here we should have an arguments, like
> 14 |   incrementActiveSounds: (value: boolean) => void

can you correct it please? :smile: